com.flashforge.FlashPrint.json

```diff
@@ -0,0 +1,81 @@
+{
+  "app-id": "com.flashforge.FlashPrint",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.9",
+  "sdk": "org.kde.Sdk",
+  "command": "FlashPrint",
+  "tags": [
+    "proprietary"
+  ],
+  "finish-args": [
+    "--filesystem=home",
+    "--socket=wayland",
```

Comment:
I guess it shouldn't specify wayland here if it's broken. `x11` should be enough to use xwayland

Reply:
You are right, the flag does not seem to have any effect. I will remove it.

Reply:
Indeed, I have now removed this flag.